products/statement-generator/src/styles/utilityStyles.ts

```diff
@@ -4,13 +4,17 @@ const useUtilityStyles = makeStyles<Theme>(
   ({ palette, breakpoints, globals, spacing }) =>
     createStyles({
       primaryContainer: {
-        minHeight: 'calc(100vh - 160px)',
+        minHeight: '100vh',
         width: '100%',
         display: 'flex',
         flex: '1 0 auto',
         flexDirection: 'column',
         position: 'relative',
 
+        [breakpoints.down('sm')]: {
+          minHeight: 'calc(100vh + 155px)',
+        },
+
```

Comment:
what is this intended to do? It currently is pushing the footer off the screen on mobile screens.

Reply:
@eburdekin let's remove the breakpoint and leave it as it was. It's not on us to make design decisions and this creates a scroll on pages that don't require it which is a UX thing.

Reply:
You can leave how you adjusted the PageContainer minHeight to 100vh in utilityStyles.ts, just remove the adjustment for sm breakpoint